### PR TITLE
switch to uuid-random instead of uuid package

### PR DESCRIPTION
**Pull request by jacklrs**
Because usage of synchronous functions inside "uuid" package performance suffers, if knex is used in context of node based web-server. I tested "uuid-random" package which increase performance by 30%. (despite of my commit comment, new package does use synchr. functions, but in optimized way)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -83,7 +83,8 @@
     "dev": "rimraf ./lib && babel -w src --out-dir lib --copy-files",
     "lint": "eslint '{src,test}/**/*.js'",
     "plaintest": "mocha --exit -t 10000 -b -R spec test/index.js && npm run tape",
-    "prepublish": "npm run babel",
+    "prepare": "npm run babel",
```
Comment (kibertoad) [inline package.json]: why was this changed?

**Comment by kibertoad** [comment]
I'm not sure about using not very battle-tested library (1k downloads/week vs 10m for uuid). Is this a hot spot and bottleneck?

**Comment by jacklrs** [comment]
I avoided any synchronous function calls so far, and this is the only one alerted by node (WARNING: Detected use of sync API ...) in my app servers log (in dev-mode) so far. The best might be, to use something which create UUIDs without usage of synchronous functions ... but did not find a package that so far ...

File: package.json (hunk 1 above)
Comment (kibertoad) [reply package.json]: This sounds out of scope for this PR, though. Could you prepare a separate PR with explanation of the change for this/

**Comment by elhigu** [comment]
I dont see any reason to change this. 30% difference in microbenchmarks is not valid reason.

If you are able to show some realworld benchmark, where the difference is significant that would be a good reason.

**Comment by Jabher** [comment]
I think you can probably use https://github.com/ai/nanoid for that (I'm not not affiliated with author)
It gives nearly same performance increase (~25%, according to docs), also it uses less memory for IDs due to the length
